cms/lib/xblock/test/test_authoring_mixin.py

```diff
@@ -27,16 +27,18 @@ def setUp(self):
             parent_location=chapter.location,
             display_name='Test Sequential'
         )
-        self.vertical = ItemFactory.create(
+        vertical = ItemFactory.create(
```

Comment:
Using the stored xblocks causes issues as they may not reflect updates (with split). So just store the locations and look up the xblocks when needed.